.golangci.yml

```
@@ -1,6 +1,10 @@
 # Copyright 2021-2023 D2iQ, Inc. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+run:
+  skip-dirs:
+    - pkg/external
```

Does this need to be?
```suggestion
    - common/pkg/external/
```

---

No, because the linter works on a per module basis, in the root module this package isn't linted, only in the common module where this is the correct path to ignore, but it shares linter config...